ports/argtable3/vcpkg.json

```
@@ -1,14 +1,11 @@
 {
   "name": "argtable3",
   "version-string": "3.3.0.116da6c",
```

It would be great if it could eventally use a regular version scheme, enabling `version>=` expressions.

---

Hi @dg0yt , if I change the version number to `3.3.0+116da6c`, which follows the semver standard and uses `+` to separate the hash value from the version number, will `version>=` work?

---

This would work:
```suggestion
  "version-semver": "3.3.0+116da6c",
```
However it seems strange to me have a "build identifier" on a source package. What I really see, and what semver would see for comparison, is
```suggestion
  "version": "3.3.0",
```
But this not what teh git tags use.

---

I see. I will change the version string back to `<major>.<minor>.<patch>` in the next release. Or do you want me to change now? If that's the case, then I may need to prepare the next release `v3.3.1`, otherwise, I won't be able to change.

---

"3.3.30" is the displayed label of the release. This port update could use `"version": "3.3.0"` now, and simply append the extra suffix in the `REF` line of the portfile. Being able to derive the `REF` from `${VERSION}` is a mere port maintenance convenience and not a requirement.

---

I'm not using the `vcpkg_from_github` function, but using the `vcpkg_download_distfile` function, so I may not have this `REF` parameter. I'll merge this PR, and then fix this version issue in the next release. Thanks for your suggestion.